backend/data/raw_data_cleaning.py

```diff
@@ -3,9 +3,7 @@
 import geopandas as gpd
```

Comment:
wondering whether this is part of @ninadicara PR or not.. is this the very same module?

Reply:
yes-deleted